File: 1131.py
x=1
grenais = 0
inter = 0
gremio = 0
empate = 0


while x == 1:
    y = input().split()
    a,b=y
    a=int(a)
    b=int(b)
    grenais = grenais + 1
    if a > b:
        inter = inter + 1
    if a < b:
        gremio = gremio + 1
    if a == b:
        empate = empate + 1
    while True:
        x = int(input('Novo grenal (1-sim 2-nao)'))
        if x == 2 or x == 1:
            break
print('{} grenais'.format(grenais))
print('Inter:{}'.format(inter))
print('Gremio:{}'.format(gremio))
print('Empates:{}'.format(empate))
if inter > gremio:
    print('Inter venceu mais')
if gremio > inter:
    print('Gremio venceu mais')
if gremio == inter:
    print('Nao houve vencedor')
    

File: CSharp/1131.cs
using System;

class URI
{
    static void Main()
    {
        int contgrenais = 0;
        int contgremio = 0;
        int continter = 0;
        int contempate = 0;

        while (true)
        {
            string[] input = Console.ReadLine().Split();
            int i = int.Parse(input[0]);
            int g = int.Parse(input[1]);

            if (i > g)
            {
                continter++;
            }
            else if (g > i)
            {
                contgremio++;
            }
            else
            {
                contempate++;
            }

            contgrenais++;
            Console.WriteLine("Novo grenal (1-sim 2-nao)");
            string resposta = Console.ReadLine();

            if (resposta == "2")
            {
                break;
            }
        }

        Console.WriteLine($"{contgrenais} grenais");
        Console.WriteLine($"Inter:{continter}");
        Console.WriteLine($"Gremio:{contgremio}");
        Console.WriteLine($"Empates:{contempate}");

        if (contgremio > continter)
        {
            Console.WriteLine("Gremio venceu mais");
        }
        else if (continter > contgremio)
        {
            Console.WriteLine("Inter venceu mais");
        }
        else
        {
            Console.WriteLine("Nao houve vencedor");
        }
    }
}
